<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>418</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../DionysusIcons.qrc">
    <normaloff>:/icons/Icon16.PNG</normaloff>:/icons/Icon16.PNG</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="settingTab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="serverTab">
      <attribute name="icon">
       <iconset resource="../DionysusIcons.qrc">
        <normaloff>:/icons/earth_network.png</normaloff>:/icons/earth_network.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Server</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="midasURLLabel">
         <property name="text">
          <string>Midas URL :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>3</number>
         </property>
         <property name="buddy">
          <cstring>midasURLEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Check server connection </string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Check email and password</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="checkServerConnectivityLabel">
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../DionysusIcons.qrc">:/icons/QuestionMark_48x48.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="checkLoginLabel">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../DionysusIcons.qrc">:/icons/QuestionMark_48x48.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="midasURLEdit">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="checkConnectionSettingsButton">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Check</string>
         </property>
         <property name="icon">
          <iconset resource="../DionysusIcons.qrc">
           <normaloff>:/icons/check2.png</normaloff>:/icons/check2.png</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="searchTab">
      <attribute name="icon">
       <iconset resource="../DionysusIcons.qrc">
        <normaloff>:/icons/find.png</normaloff>:/icons/find.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Search</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="serachModeBox">
         <property name="title">
          <string>Search Mode</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="searchInAllWordRadioButton">
            <property name="text">
             <string>All words</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="searchInAtLeastRadioButton">
            <property name="text">
             <string>At least one word</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="searchInBox">
         <property name="title">
          <string>Search in :</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="searchInTitlesCheckBox">
            <property name="text">
             <string>Titles</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="searchInKeywordsCheckBox">
            <property name="text">
             <string>Keywords</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="searchInAbstractsCheckBox">
            <property name="text">
             <string>Abstracts</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="searchInSponsorsCheckBox">
            <property name="text">
             <string>Sponsors</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="searchInAuthorsCheckBox">
            <property name="text">
             <string>Authors</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="searchInDescriptionsCheckBox">
            <property name="text">
             <string>Descriptions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="searchInBitstreamsCheckBox">
            <property name="text">
             <string>Bitstreams</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>searchInAllWordRadioButton</tabstop>
  <tabstop>searchInAtLeastRadioButton</tabstop>
  <tabstop>searchInTitlesCheckBox</tabstop>
  <tabstop>searchInKeywordsCheckBox</tabstop>
  <tabstop>searchInAbstractsCheckBox</tabstop>
  <tabstop>searchInSponsorsCheckBox</tabstop>
  <tabstop>searchInAuthorsCheckBox</tabstop>
  <tabstop>searchInDescriptionsCheckBox</tabstop>
  <tabstop>searchInBitstreamsCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="../DionysusIcons.qrc"/>
  <include location="../DionysusIcons.qrc"/>
  <include location="../DionysusIcons.qrc"/>
  <include location="../DionysusIcons.qrc"/>
  <include location="../DionysusIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
